packages/medusa/src/api-v2/admin/stores/route.ts

```diff
@@ -0,0 +1,44 @@
+import { createStoresWorkflow } from "@medusajs/core-flows"
+import { CreateStoreDTO } from "@medusajs/types"
+import { remoteQueryObjectFromString } from "@medusajs/utils"
+import { MedusaRequest, MedusaResponse } from "../../../types/routing"
+import { defaultAdminStoreFields } from "./query-config"
+
+export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
+  const remoteQuery = req.scope.resolve("remoteQuery")
+
+  const queryObject = remoteQueryObjectFromString({
+    entryPoint: "store",
+    variables: {
+      filters: req.filterableFields,
+      order: req.listConfig.order,
+      skip: req.listConfig.skip,
+      take: req.listConfig.take,
+    },
+    fields: defaultAdminStoreFields,
+  })
+
+  const { rows: stores, metadata } = await remoteQuery(queryObject)
+
+  res.json({
+    stores,
+    count: metadata.count,
+    offset: metadata.skip,
+    limit: metadata.take,
+  })
+}
+
+export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
+  const input = [req.validatedBody as CreateStoreDTO]
+
+  const { result, errors } = await createStoresWorkflow(req.scope).run({
+    input: { storesData: input },
```

Comment:
todo: As per our latest discussion, workflow input should be snake_cased. Or  alternatively renamed:
```suggestion
    input: { stores: input },
```